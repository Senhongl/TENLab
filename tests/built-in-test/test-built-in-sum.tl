a = [2,2,3];
print(sum(a));
b = [2.0,2.2,3.5];
print(sum(b));