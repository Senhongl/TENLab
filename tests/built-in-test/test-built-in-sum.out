7
[ CPUIntType{} ]
7.7
[ CPUDoubleType{} ]